syncto/tests/test_authentication.py

```diff
@@ -33,10 +39,29 @@ def setUp(self):
     def test_should_raise_if_authorization_header_is_missing(self):
         self.assertRaises(HTTPUnauthorized, build_sync_client, self.request)
 
-    def test_should_raise_if_client_state_header_is_missing(self):
+    def test_should_raise_if_client_state_header_is_missing_with_syncto(self):
         self.request.headers = {AUTHORIZATION_HEADER: 'Browserid 1234'}
         self.assertRaises(HTTPUnauthorized, build_sync_client, self.request)
 
+    def test_should_not_raise_if_client_state_in_url(self):
```

Comment:
nit: I would say that the test goes beyond *not raising*. It checks that the bucket id is used as the client state for sync client